LabExT/Exporter/Formats/ExportJSON.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2023  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+
+import json
+from os.path import join, exists
+from pathlib import Path
+
+from LabExT.Exporter.ExportStep import ExportFormatStep
+
+class ExportJSON(ExportFormatStep):
```

Comment:
JSON exporter unnecessary as LabExT by default saves all measurements in JSON, can be removed.

Reply:
This has been removed.